tgui/packages/tgui/interfaces/ERTManager.js

```diff
@@ -33,6 +34,17 @@ export const ERTManager = (props, context) => {
                 onClick={() => act('ert_type', { ert_type: 'Gamma' })}
               />
             </LabeledList.Item>
+          <LabeledList.Item label="Loadout">
+              {data.loadout_types.map((loadout) => (
+                <Button
+                  key={loadout.path}
+                  content={loadout.name}
+                  textAlign="center"
+                  color={data.chosen_loadout === loadout.path ? 'green' : ''}
+                  onClick={() => act('chosen_loadout', { chosen_loadout: loadout.path })}
```

Comment:
Can you adjust the indentations for these button props, please?

Reply:
Adjust how?

Reply:
Sorry I forgot to respond. Looking at this again, could you indent the element on line 37 once to the right so that it lines up properly with the rest of the elements below it?